integration-webcert/src/main/java/se/inera/intyg/minaintyg/integration/webcert/converter/metadata/StatusConverter.java

```diff
@@ -0,0 +1,87 @@
+package se.inera.intyg.minaintyg.integration.webcert.converter.metadata;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.function.Predicate;
+import java.util.stream.Stream;
+import org.springframework.stereotype.Component;
+import se.inera.intyg.minaintyg.integration.api.certificate.model.common.CertificateStatusType;
+import se.inera.intyg.minaintyg.integration.webcert.client.dto.CertificateMetadataDTO;
+import se.inera.intyg.minaintyg.integration.webcert.client.dto.metadata.CertificateRelation;
+import se.inera.intyg.minaintyg.integration.webcert.client.dto.metadata.CertificateRelationType;
+
+@Component
+public class StatusConverter {
+
+  public static final int DAYS_LIMIT_FOR_STATUS_NEW = 14;
```

Comment:
An idea! If you move this constant to the integration-api package.. you can then use it in both the webcert and intygstjanst implementations.